package.json

```diff
@@ -9,32 +9,32 @@
     "url": "https://github.com/juju/juju-gui.git"
   },
   "dependencies": {
-    "classnames": "2.2.5",
-    "clipboard": "1.7.1",
+    "classnames": "^2.2.6",
+    "clipboard": "^1.7.1",
```

Comment:
Didn't we decide for not using ^ in deps?

Reply:
Yeah I can go through and remove those, however npm now installs from the package-lock so even though this has ^, it'll still only install the version specified in the lock.

Reply:
I've never seen it respect the package lock, but maybe that's a new thing...